params/extras/config_test.go

```diff
@@ -0,0 +1,216 @@
+// (c) 2025 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package extras
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/upgrade"
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/subnet-evm/commontype"
+	"github.com/ava-labs/subnet-evm/precompile/contracts/txallowlist"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func pointer[T any](v T) *T { return &v }
+
+func TestChainConfigDescription(t *testing.T) {
```

Comment:
This is just a change-detector test that IMO not only adds no value, results in maintenance burden because any tiny change to the hook just has to be mirrored here. If it's absolutely necessary to check that the hook is working (I'm skeptical but open to convincing) then just test for the presence of a substring like "SubnetEVM".

Reply:
We're not really testing FeeConfig in particular, and there is no hook here.
However, the part checking the network upgrades will have to be changed everytime a new upgrade timestamp is set in the `NetworkUpgrades` struct, so that's a change detector issue: test was changed to use a regex instead with N timestamp lines, so that would adapt to future timestamps.

The point of the test is to detect a missing piece in the description method (such as the feeconfig); so we need to check *most* of the string, not just that it contains 1 element, such that this won't happen for other fields used in the description.